packages/components/tests/dummy/app/templates/components/modal.hbs

```diff
@@ -270,30 +273,27 @@
 
   {{! template-lint-disable no-autofocus-attribute }}
   {{#if this.formModalActive}}
-    <Hds::Modal id="form-modal" @onClose={{fn this.deactivateModal "formModalActive"}} as |M|>
+    <Hds::Modal id="form-modal" @onClose={{fn this.checkBeforeDeactivate "formModalActive"}} as |M|>
       <M.Header>
         Why do you want to leave the beta?
       </M.Header>
       <M.Body>
-        <Hds::Form::Select::Field autofocus @width="100%" as |F|>
-          <F.Label>Select the primary reason</F.Label>
-          <F.Options>
-            <option></option>
-          </F.Options>
-        </Hds::Form::Select::Field>
-        <Hds::Form::Textarea::Field @isOptional={{true}} as |F|>
-          <F.Label>Your feedback</F.Label>
-        </Hds::Form::Textarea::Field>
+        <form {{on "change" this.markFormAsChanged}} name="leaving-beta-form">
+          <Hds::Form::Select::Field autofocus @width="100%" as |F|>
+            <F.Label>Select the primary reason</F.Label>
+            <F.Options>
+              <option></option>
+            </F.Options>
+          </Hds::Form::Select::Field>
+          <Hds::Form::Textarea::Field @isOptional={{true}} as |F|>
+            <F.Label>Your feedback</F.Label>
+          </Hds::Form::Textarea::Field>
+        </form>
       </M.Body>
-      <M.Footer>
+      <M.Footer as |F|>
         <Hds::ButtonSet>
           <Hds::Button type="submit" @text="Leave Beta" />
-          <Hds::Button
-            type="submit"
-            @text="Cancel"
-            @color="secondary"
-            {{on "click" (fn this.deactivateModal "formModalActive")}}
-          />
+          <Hds::Button type="submit" @text="Cancel" @color="secondary" {{on "click" F.close}} />
```

Comment:
```suggestion
          <Hds::Button type="button" @text="Cancel" @color="secondary" {{on "click" F.close}} />
```
You'd really only want one submit button in a form- you could use `reset` as the value but in general I don't see that used anymore.

Reply:
ah, yes, good spot! – copy/paste habits, will update to `button`

Reply:
they were all over the examples 🙀